applications/API/filters.py

```
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+from django_filters import rest_framework as filters
+
+from .models import Insult
```

Pylint (3.3.7)

[error] Cannot import 'models' due to 'expected 'except' or 'finally' block (git.applications.API.models, line 231)'

(E0001)